components/feature/tabs/src/main/java/mozilla/components/feature/tabs/TabsUseCases.kt

```diff
@@ -217,6 +217,20 @@ class TabsUseCases(
         }
     }
 
+    /**
+     * Use case for removing a list of tabs.
+     */
+    class RemoveSelectedTabsUseCase internal constructor(
+        private val sessionManager: SessionManager
+    ) {
+        /**
+         * Removes a specified list of tabs.
+         */
+        operator fun invoke(sessionList: List<Session>) {
```

Comment:
Ideally, if possible, we pass in the list of `String` ids here (and to `SessionManager`). This way we can redirect it to `BrowserStore` eventually, here and the code that calls this, without having to deal with `Session`s.